bin/check-sensor.rb

```diff
@@ -86,6 +86,12 @@ class CheckSensor < Sensu::Plugin::Metric::CLI::Graphite
          long: '--ipmitool IPMI_PROVIDER',
          default: 'ipmitool'
 
+  option :driver,
+         description: 'IPMI Tool Driver: auto, lan15, lan20, open (defaults to lan20)',
+         short: '-d IPMI_DRIVER',
+         long: '--driver IPMI_DRIVER',
+         default: 'lan20'
+
```

Comment:
Let's add the list of allowed like this: https://github.com/sensu-plugins/sensu-plugins-redis/blob/2.3.0/bin/check-redis-info.rb#L67

Reply:
In case you wanted to see the source: https://github.com/chef/mixlib-cli/blob/v1.7.0/lib/mixlib/cli.rb#L240-L250